cnames_active.js

```diff
@@ -1819,6 +1819,7 @@ var cnames_active = {
   "liveflow": "hasharray.github.io/liveflow.js",
   "livenotif": "siffreinsg.github.io/livenotif",
   "livevalidate": "thedhanawada.github.io/LiveValidateJS",
+  "lizzy": "https://stoyan.github.io/Lizzy.js/",
```

Comment:
```suggestion
  "lizzy": "stoyan.github.io/Lizzy.js",
```
Per the CI error, you do not need the leading https://, nor the trailing /